workloads/virtual-machines/interfaces-and-networks.md

```diff
@@ -62,9 +62,10 @@ In some cases the underlying network plugin (flannel, weave, OpenShift SDN) acts
 
 ## Available interface attributes
 
-|Name|Possible values|Default|Description|
+|Name|Format|Default|Description|
 |--|--|--|--|
-|model|e1000, e1000e, ne2k_pci, pcnet, rtl8139, virtio|virtio|Interface model type exposed to guest (tip: use e1000 if your image doesn't support virtio)|
+|model|One of: e1000, e1000e, ne2k_pci, pcnet, rtl8139, virtio|virtio|Interface model type exposed to guest (tip: use e1000 if your image doesn't support virtio)|
+|macAddress|ff:ff:ff:ff:ff:ff or FF-FF-FF-FF-FF-FF||MAC address as seen inside the guest system|
```

Comment:
Please add an example.

Reply:
Not sure if that's what you mean, but I added one in description. The format column already gave enough context though so not sure if that's what you were asking.

Reply:
I think @fabiand is asking to update the yaml example on line 31. Perhaps we had to update it with the introduction of the `model` attribute.

Reply:
Looks good to me, besides the yaml exmaple update.

Reply:
OK gotcha; I've added macAddress (and model) there. I haven't touched e.g. delegateIP that doesn't really exist for now. This is belong to a different PR.